Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx

```diff
@@ -358,12 +358,16 @@ export const ObiEditor: FC<ObiEditorProps> = ({
           case KeyboardCommandTypes.Node.Cut:
             dispatchEvent(NodeEventTypes.CutSelection, { actionIds: getClipboardTargetsFromContext() });
             break;
-          case KeyboardCommandTypes.Node.Paste:
-            dispatchEvent(NodeEventTypes.AppendSelection, {
-              target: focusedId,
-              actions: clipboardActions,
-            });
+          case KeyboardCommandTypes.Node.Paste: {
+            const currentSelectedId = selectionContext.selectedIds[0];
+            if (currentSelectedId.endsWith('+')) {
+              dispatchEvent(NodeEventTypes.AppendSelection, {
+                target: focusedId,
+                actions: clipboardActions,
+              });
```

Comment:
Do we want there to be any kind of feedback when a user tries to paste something into an invalid place?

Reply:
We do not have any feedback at least. Should we pop a popover to show these feedback? @cwhitten  @beyackle

Reply:
I don't see why not. @alanlong9278 how about:

> "paste operation not allowed at current element"